Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -249,6 +249,15 @@ theorem card' : ∃ (p : ℕ) (n : ℕ+), Nat.Prime p ∧ Fintype.card K = p ^ (
   let ⟨p, hc⟩ := CharP.exists K
   ⟨p, @FiniteField.card K _ _ p hc⟩
 
+omit [Fintype K] in
+lemma two_le_card [Finite K] : 2 ≤ Nat.card K := by
+  haveI : Fintype K := Fintype.ofFinite K
+  obtain ⟨p, n, h⟩ := FiniteField.card' K
+  rw [← Fintype.card_eq_nat_card, h.right, ← pow_one 2]
+  exact le_trans
+    (Nat.pow_le_pow_of_le (by simp only [Nat.one_lt_ofNat]) NeZero.one_le)
+    (Nat.pow_le_pow_of_le_left (Nat.Prime.two_le h.left) n)
```

Comment:
(untested)

```suggestion
lemma two_le_card [Finite K] : 2 ≤ Nat.card K := by
  Finite.one_lt_card
```

This should work for any type which is `Finite` and `Nontrivial` (which automatically holds for `Field`). Maybe you want to add `Finite.two_le_card` instead.

Reply:
Indeed, this lemma is just nonsense, I don't know what I was thinking. Thanks!